### DISCUSSION TITLE
Cannot determine type of "select"  [has-type]mypy(error)

### DISCUSSION THREAD

**Discussion by toinbis** [sqlalchemy/sqlalchemy#7235] [Usage Questions]
Hey all!

So I have code:
`query: Select = select(table.c.id, table.c.name)`

Mypy shows an error for this line: `Cannot determine type of "select"  [has-type]mypy(error)`

Any ideas on why is the error appearing and what to do to satisfy mypy?

Thanks!

**Comment by CaselIT** [comment]
Hi,

Could you provide what version you are using? Both sqlalchemy and the stubs.
